tools/api-markdown-documenter/package.json

```diff
@@ -88,7 +88,7 @@
 		"@fluid-tools/markdown-magic": "file:../markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.44.0",
-		"@fluidframework/eslint-config-fluid": "^5.1.0",
+		"@fluidframework/eslint-config-fluid": "^5.5.1",
```

Comment:
We've been doing a minor release on `eslint-config-fluid` if introducing a custom linter rule (so also a bump on `eslint-plugin-fluid`) right? 

If I am remembering correctly, it's currently at `5.5.1` since the latest release was fixing the `minimal.js`.

Reply:
This package doesn't depend directly on the plugin package - no bump other dep bump should be needed. The appropriate version of the plugin will get installed automatically via the config dep.